src/containers/app/index.js

```diff
@@ -6,17 +6,19 @@ import Body from '../body'
 import Header from '../header'
 import Sidebar from '../sidebar'
 import Chat from '../chat'
+import RegisterStepTwo from '../registerStepTwo'
 
 const App = () => (
     <div>
         <main>
             <Route exact path="/" component={Login} />
-            <Route exact path="/login" component={Login} />
+            <Route exact path="/login.js" component={Login} />

```

Comment:
Is it needed to have a ".js" extension, didn't it work previously without it?

Reply:
No it doesn't need it, so changing it to just "/login" thanks.